glific/floweditor
# Feature/multilingual attachments

## Summary
Attachments tab added for translated node

### Steps to test
1. In package.json change this   "@nyaruka/flow-editor": "glific/floweditor#v1.11.8.0" to   "@nyaruka/flow-editor": "glific/floweditor#feature/multilingual-attachments"
2. Run `yarn install && yarn run floweditor`
3. Restart frontend server
4. Finally check if the attachment support is there the other language dialog box in glific-frontend
